### PR TITLE
feat: 🎸 Add details for RData

### PR DESCRIPTION
**Pull request by yangliu2009**
Added Location, Rates, Access and Transfer for RData

### Pull Request
---

#### PR Summary
Briefly describe the changes.

#### Check the types of changes present in this PR:
- [ ] :bug: Bug
- [ ] :dragon: Feature
- [ ] :frog: Data (data folder - people/opportunities)
- [x] :dog: Content (content folder)
- [ ] :blowfish: Other. Specify:

#### Checklist:
- [x ] Commitizen was used for all commits.
- [x ] Local site looks as expected after changes.
- [ x] Contributing guidelines were followed.
- [ ] If changes affect development, was the README updated?

##### If PR to `production`
- [ ] Development site (https://datasci.brown.edu) was checked and looks as expected.

### CODE REVIEW

**Comment by paulstey** [comment]
This looks great; thank you, @yangliu2009!!!

**Review by paulstey — APPROVED** [review]

File: content/english/services/infrastructure/rdata.md [hunk 1]
```diff
@@ -12,4 +12,21 @@ icon: server2
...
+# Rates
+Here are the details of [Rates for Research Data Storage](https://ccv.brown.edu/services/rates/)
+
+# Access
+* Users can access their RData by [ssh](https://docs.ccv.brown.edu/oscar/connecting-to-oscar/ssh) or [VNC](https://docs.ccv.brown.edu/oscar/connecting-to-oscar/vnc) into Oscar.
+* Users can access their RData as a local mount on their computers using either [SMB](https://docs.ccv.brown.edu/oscar/connecting-to-oscar/cifs) or [CIFS](https://docs.ccv.brown.edu/oscar/connecting-to-oscar/cifs-non-ad-local-mount)
+* Users can access the [snapshops of their RData](https://docs.ccv.brown.edu/oscar/managing-files/restore) for the last 5-7 days.
+
+# Transfer
+* Users can [transfer data between their local computers and Rdata](https://docs.ccv.brown.edu/oscar/managing-files/filetransfer-isilon)
```
Comment (mirestrepo) [inline content/english/services/infrastructure/rdata.md]: I think the link should be https://docs.ccv.brown.edu/oscar/managing-files/filetransfer

**Review by mirestrepo — CHANGES_REQUESTED** [review]

File: content/english/services/infrastructure/rdata.md (hunk 1 above)
Comment (yangliu2009) [reply content/english/services/infrastructure/rdata.md]: nice catch, Isabel. The link is corrected now. Thanks.

**Comment by mcmcgrath13** [comment]
@yangliu2009 I changed the pr type to `content` as this just touches the content folder - less review needed 😉

**Review by mcmcgrath13 — APPROVED** [review]

**Pull request merged**